src/components/vm/snapshots/vmSnapshotsCreateModal.jsx

```diff
@@ -127,15 +127,15 @@ export class CreateSnapshotModal extends React.Component {
         this.setState({ dialogError: text, dialogErrorDetail: detail });
     }
 
-    onValidate(submitted = false) {
+    onValidate() {
```

Comment:
Real minor nit, there is a typo in the commit message `should not bet empty`

Reply:
Another minor nit and something we do super inconsistent in Cockpit is setting `isRequired` on the TextInput so the dialog shows a hint what is optional or not.

![image](https://github.com/cockpit-project/cockpit-machines/assets/67428/c608461d-8943-422c-beae-2266ac07452d)

Reply:
We only use it in one place in cockpit (kdump dialog), once in podman (publish port), and not at all in c-machines. So perhaps right now it's best to not start with that in c-machines, but rather do that consistently on all dialogs, in a separate PR?